bears/c_languages/GNUIndentBear.py

```diff
@@ -23,18 +23,167 @@ class GNUIndentBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=80,
                          use_spaces: bool=True,
+                         blank_lines_after_declarations: bool=False,
+                         blank_lines_after_procedures: bool=False,
+                         blank_lines_after_commas: bool=False,
+                         braces_on_if_line: bool=False,
+                         braces_on_func_def_line: bool=False,
+                         cuddle_else: bool=False,
+                         while_and_brace_on_same_line: bool=False,
+                         case_indentation: int=0,
+                         space_before_semicolon_after_empty_loop: bool=True,
+                         delete_optional_blank_lines: bool=True,
+                         declaration_indent: int=0,
+                         brace_indent: int=2,
+                         gnu_style: bool=False,
+                         k_and_r_style: bool=False,
+                         linux_style: bool=False,
                          tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
                          indent_cli_options: str=''):
         """
-        :param max_line_length:    Maximum number of characters for a line.
-        :param use_spaces:         True if spaces are to be used, else tabs.
-        :param tab_width:          Number of spaces per indent level.
-        :param indent_cli_options: Any command line options the indent binary
-                                   understands. They will be simply passed
-                                   through.
+        :param max_line_length:
+            Maximum number of characters for a line.
+        :param use_spaces:
+            True if spaces are to be used, else tabs.
+        :param blank_lines_after_declarations:
+            Forces blank lines after the declarations.
+
+            Example:
+                ```
+                If True,
```

Comment:
ok, so i got the true meaning of this line now, it states if the is setting is True and the rest of the code is after that, can you avoid this confusion? maybe you should put the IF Tue before the example or maybe reword its so that it is simpler, sorry for the confusion.